Specs/spec-main.js

```diff
@@ -136,6 +136,8 @@
 
         window.it = function(description, f, timeout, categories) {
             originalIt(description, function(done) {
+                // Clear the RequestScheduler before for each test in case requests are still active from previous tests
+                Cesium.RequestScheduler.clearForSpecs();
```

Comment:
There could be cases where unrelated tests do not resolve their promises correctly so the `activeRequests` list can get clogged. But since this only affects throttled requests the `clearForSpecs` only need to go in terrain/imagery provider tests.